packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/index.tsx

```diff
@@ -331,13 +346,49 @@ const CommentBody = ({ comment, editing, setEditing }) => {
   );
 };
 
-const Replies = ({ replies }) => (
-  <>
-    {replies.map(reply =>
-      reply ? <Reply reply={reply} key={reply.id} /> : 'Loading...'
-    )}
-  </>
-);
+const Replies = ({ replies, repliesRenderedCallback }) => {
+  const [repliesLoaded, setRepliesLoaded] = React.useState(!!replies[0]);
```

Comment:
Do we need this? Can just use `const repliesLoaded = !!replies[0]`?

Reply:
Ah, we might not need it anymore, will check!

Reply:
Done!